kafka_consumer/tests/docker/multiple-brokers.yaml

```diff
@@ -2,40 +2,47 @@ version: '3.5'
 
 services:
   zookeeper:
-    image: zookeeper:${ZK_VERSION}
+    image: docker.io/bitnami/zookeeper:${ZK_VERSION}
     ports:
       - 2181:2181
+    environment:
+      - ALLOW_ANONYMOUS_LOGIN=yes
 
   kafka1:
-    image: "wurstmeister/kafka:${KAFKA_VERSION}"
-    hostname: kafka1
+    image: docker.io/bitnami/kafka:${KAFKA_VERSION}
     ports:
       - 9092:9092
     environment:
-      KAFKA_BROKER_ID: 1
-      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
-      KAFKA_CREATE_TOPICS: "marvel:2:1,dc:2:1"
-      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "true"
-      KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
-      KAFKA_LISTENERS: INTERNAL://:19092,EXTERNAL://:9092
-      KAFKA_ADVERTISED_LISTENERS: INTERNAL://:19092,EXTERNAL://:9092
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
-
+      KAFKA_CFG_BROKER_ID: 1
+      KAFKA_CFG_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_CFG_INTER_BROKER_LISTENER_NAME: INTERNAL
+      KAFKA_CFG_LISTENERS: INTERNAL://:19092,EXTERNAL://:9092
+      KAFKA_CFG_ADVERTISED_LISTENERS: INTERNAL://:19092,EXTERNAL://127.0.0.1:9092
+      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+      ALLOW_PLAINTEXT_LISTENER: "true"
     depends_on:
       - zookeeper
 
   kafka2:
-    image: "wurstmeister/kafka:${KAFKA_VERSION}"
-    hostname: kafka2
+    image: docker.io/bitnami/kafka:${KAFKA_VERSION}
     ports:
       - 9093:9093
     environment:
-      KAFKA_BROKER_ID: 2
-      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
-      KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
-      KAFKA_LISTENERS: INTERNAL://:19093,EXTERNAL://:9093
-      KAFKA_ADVERTISED_LISTENERS: INTERNAL://:19093,EXTERNAL://:9093
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+      KAFKA_CFG_BROKER_ID: 2
+      KAFKA_CFG_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_CFG_INTER_BROKER_LISTENER_NAME: INTERNAL
+      KAFKA_CFG_LISTENERS: INTERNAL://:19093,EXTERNAL://:9093
+      KAFKA_CFG_ADVERTISED_LISTENERS: INTERNAL://:19093,EXTERNAL://127.0.0.1:9093
+      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+      ALLOW_PLAINTEXT_LISTENER: "true"
 
     depends_on:
       - zookeeper
+
+  kafka-init:
+    image: docker.io/bitnami/kafka:${KAFKA_VERSION}
+    command: >
+      sh -c "sleep 10 &&./opt/bitnami/kafka/bin/kafka-topics.sh ${BOOTSTRAP_SERVER_FLAG} --create --topic marvel --if-not-exists --partitions 2 --replication-factor 1 &&
```

Comment:
Why `sleep 10`?

Reply:
It's to wait for the broker to be ready otherwise, the create topics command will timeout too soon